src/Angor/Client/Pages/Invest.razor

```diff
@@ -677,7 +678,7 @@ else
                 var insert = new StageBreakdown
                     {
                         StageNumber = index++,
-                        Amount = Investment.InvestmentAmount * (stage.AmountToRelease / 100),
+                        Amount = Investment.InvestmentAmountBtc * (stage.AmountToRelease / 100),
```

Comment:
once we convert to sats we should only use sats unless there is a rason you use the btc amount

Reply:
hmmm not sure about this